src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -182,20 +182,32 @@ public void register() {
 	public void register(boolean slimefun) {
 		addon = !slimefun;
 		try {
+			// Check if this item must be handled by Slimefun or return to its vanilla behavior
+			if(vanilla) SlimefunStartup.getItemCfg().setDefaultValue(this.name + ".ignore-slimefun-modifications", false);
+			if(SlimefunStartup.getItemCfg().getBoolean(this.name + ".ignore-slimefun-modifications")){
```

Comment:
The "vanilla" boolean is here to add the line to the VanillaItem(s).
Well, I can do it differently using `this instanceof VanillaItem`. Let me change that.